src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1,4 +1,31 @@
-// Add the required types and props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
-);
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+import { UserInfo } from '../UserInfo';
+
+type TodoInfoProps = {
```

Comment:
while describing props interface, you don't have to come up with a unique name
```suggestion
type Props = {
```